src/netius/base/common.py

```
@@ -4330,6 +4330,18 @@ def _ssl_destroy(self):
         self._ssl_secure = None
         self._ssl_context_options = None
 
+    def _ssl_reload(self):
+        # verifies if the current SSL contexts map supports
+        # reload (eg: LetsEncryptDict) and if that's the case
+        # triggers a reload to pick up new certificates from
+        # the file system without requiring a full restart
+        if not hasattr(self._ssl_contexts, "reload"):
+            return
+        domains = list(self._ssl_contexts.keys())
+        changed = self._ssl_contexts.reload(domains)
```

Pylint (4.0.5)

[error] Instance of 'dict' has no 'reload' member

(E1101)